src/test/java/ScattTest.java

```diff
@@ -123,6 +123,28 @@ public File getDirectoryFromUser() {
             fail("You need to put a file called " + testDir.getName() + "_Report.txt containing" 
                  + " the expected report contents in the test resources directory");
         }
+        if (!actual.equals(expected)) {
```

Comment:
It seems this would be good to include with the test utils as we might want to reuse this in other tests. As it is a nice diff feature.